OxfordBodleian/Juel-Jensen/BDLaethd20.xml

```diff
@@ -51,11 +3473,18 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                             <supportDesc>
                                 <support>
                                     <material key="parchment"/>
+                                    <p>Some folia have small holes, presumably from parchmenting: <locus
```

Comment:
"parchmenting"is this the right term?

Reply:
I know that Sean Winslow is using this term. Therefore I also use it :)

Reply:
I am not convinced, a search on scholar provides less than a dozen attestations: https://scholar.google.it/scholar?hl=en&as_sdt=0%2C5&q=%22parchmenting%22+manuscript&btnG=

Reply:
We can use "parchment making" as in ligatus (https://www.ligatus.org.uk/lob/concept/4962)